### ISSUE TITLE
Game hangs at "Creating game window with window mode Fullscreen" after updating from 1.18.8 -> 1.18.10

### ISSUE THREAD

**Issue by ca1f** [anegostudios/VintageStory-Issues#3080]
### Game Version

1.18.10

### Platform

Linux

### Modded

None

### SP/MP

None

### Description

After updating the game from 1.18.8 to 1.18.10, the game isn't launching for me anymore.

### How to reproduce

1. Update vintage story via yay from 1.18.8 to 1.18.10 on arch linux.
2. Try launching vintagestory from the command line
3. observe vintage story hanging at start with a black window

### Logs

```
~/.config/VintagestoryData/Logs $ cat client-*
4.9.2023 20:50:56 [Debug] Creating game window with window mode Fullscreen
4.9.2023 20:50:56 [Notification] Client logger started.
4.9.2023 20:50:56 [Notification] Game Version: v1.18.10 (Stable)
```

**Comment by ca1f** [comment]
So I did a little bit of research and it seems like the version 1.18.8 still launches fine. The version 1.18.10 does not launch on a system with an AMD GPU or an nvidia GPU though but would launch fine on my intel NUC.

**Comment by SamDeNies** [comment]
I have the same issue with the 1.18.10 flatpak on ArchLinux.  I have an AMD graphics card.  My logs are the same, and it hangs on a black window at the very start.

**Comment by SamDeNies** [comment]
This may be of help.  There are more messages before this, but they don't seem relevant.

Sep 06 20:31:46 yarrow systemd[1264]: Started app-flatpak-at.vintagestory.VintageStory-13456.scope.
Sep 06 20:31:48 yarrow at.vintagestory.VintageStory.desktop[13470]: 6.9.2023 20:31:48 [Client Notification] Client logger started.
Sep 06 20:31:48 yarrow at.vintagestory.VintageStory.desktop[13470]: 6.9.2023 20:31:48 [Client Notification] Game Version: v1.18.10 (Stable)
Sep 06 20:31:48 yarrow dleyna-renderer-service[13445]: dLeyna: Exit
Sep 06 20:31:48 yarrow at.vintagestory.VintageStory.desktop[13470]: 6.9.2023 20:31:48 [Client Debug] Creating game window with window mode Normal
Sep 06 20:31:48 yarrow gnome-shell[1386]: Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a00009
Sep 06 20:31:55 yarrow systemd[1264]: app-flatpak-at.vintagestory.VintageStory-13456.scope: Consumed 7.511s CPU time.

**Comment by SamDeNies** [comment]
Vintage Story is launching properly with 1.18.14 flatpak on archlinux.  I also updated my OS, so don't know if it was resolved by the game update or a dependency update.

**Comment by bluelightning32** [comment]
My laptop is running Fedora Linux 38. With the 6.4.12-200.fc38.x86_64 kernel version, I could launch Vintage Story 1.18.0, but Vintage Story 1.18.14 hung as described in this bug. The video card is an integrated AMD one: Lucienne c1.

I ran a system update, and that fixed it. I don't know which package upgrade fixed it. I did keep track of the kernel version. Now I'm running 6.5.5-200.fc38.x86_64, and it works.

**Comment by Th3Dilli** [comment]
Are you all running in Wayland? there is an issue in OpenTK where on wayland it would just freeze the window

I today made a fix for this on OpenTK and this should be in for a soon 1.19

The issue was introduced with Vintagestory 1.18.10 so the last working one should be 1.18.8

**Comment by Th3Dilli** [comment]
there was an issue related to wayland which was resolved in 1.19
